template/Makefile

```diff
@@ -1,5 +1,5 @@
 LIBDIR := lib
-include $(LIBDIR)/main.mk
```

Comment:
This isn't that simple, sadly.

I do like this change, but I'll need to tweak update.mk to match.  It's a bit involved, sadly.

Reply:
A tragedy in 3 acts
* "oh how hard could it be?"
* _checks update.mk_
* _slowly backs away from keyboard_

Reply:
😁  Leave it with me.  I've got a lot of work right now, but I know how to drive that.

Reply:
Thanks! This is incredibly low impact and priority so no rush :-)